src/test/daml/Daml/Finance/Account/Test/Util.daml

```diff
@@ -0,0 +1,25 @@
+module Daml.Finance.Account.Test.Util where
+
+import DA.Set qualified as S (empty, fromList, singleton)
+import Daml.Finance.Interface.Account.Account qualified as Account (Controllers(..))
+
+-- | Options for transfer controllers.
+data ControlledBy
+  = Owners
+    -- ^ Owners-controller.
```

Comment:
```suggestion
    -- ^ Owner controls inbound and outbound transfers.
```

I wonder if there is a more descriptive name to convey the difference between `Owner` and `Owners`